dist/bin/no-internal-report.js

```diff
@@ -1,64 +1,44 @@
 #! /usr/bin/env node
-
 /*---------------------------------------------------------------------------------------------
 * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
 * See LICENSE.md in the project root for license terms and full copyright notice.
 *--------------------------------------------------------------------------------------------*/
 "use strict"
 const path = require('path');
-const fs = require('fs');
-
-function getNodeModulesDir(startPath) {
-  let currPath = startPath;
-  let baseName = path.basename(currPath);
-  while (baseName !== "node_modules") {
-    currPath = path.resolve(currPath, "..")
-    if (currPath === "/")
-      return undefined;
-    baseName = path.basename(currPath);
+const assert = require('assert');
+const eslintBinPath = path.join(require.resolve("eslint"), "../../bin/eslint.js");
+
+const args = { tags: ["alpha", "internal"], files: [] };
+for (let i = 2; i < process.argv.length; ++ i) {
+  if (process.argv[i] == '--tags') {
+    assert(i + 1 < process.argv.length, "tags option requires an argument");
+    args.tags = process.argv[i + 1].split(",");
+    i += 1; // skip next arg
+  } else if(process.argv[i] == '--files') {
+    assert(i + 1 < process.argv.length, "files option requires an argument");
+    args.files.push(process.argv[i + 1]);
+    i += 1; // skip next arg
```

Comment:
Since it looks like both happy paths do this, instead of doing `i+=1` perhaps you should make your for loop statement: 
```ts
for (let i = 2; i < process.argv.length; i+=2) {...}
```

Reply:
changes made allow multiple file globs. Leaving the iterator +1 so we dont skip any or go out of bounds

Reply:
For for loops the second statement, i.e. `i < process.argv.length` check happens before the beginning of an iteration and the third statement `++i` happens at the end of any iteration. This means if `i ` goes out of bounds at the end of an iteration, the `i < process.argv.length` check before start of next iteration returns `false` and the process breaks out of the for-loop insuring no illegal out of bound element access attempt.

Maybe someone else can comment on if one approach is readable than the other but probably not a huge deal all in all

Reply:
I see that this no longer applies due to the more recent multiple file globs accepting implementation. resolving now, thanks